src/C-DEngine/C-DCommunication/cde_QueuedSenderRegistry.cs

```diff
@@ -989,6 +998,22 @@ internal static bool DoPublish(string pTopic, TSM pMsg,bool AddScopeToTopic, Act
             return false;
         }
 
+        static char[] wildcard = {'*'};
+        private static bool MatchTSMFilter(TSM pMsg, string[] TSMFilter)
+        {
+            bool match = false;
```

Comment:
Maybe here a quick out if (pMSG==null || TSMFilter==null) return false;

Reply:
Actually how about making this internal or even public in TheCommonUtils? I can see this being used for other reasons as well - even by plugins that want to parse TSM.TXT with filters

Reply:
I'd rather keep it private for now. It's not a full regex validator, only works for a single * etc. That's why it doesn't have the quick out check either.